public/apps/configuration/panels/internal-user-edit/internal-user-edit.tsx

```diff
@@ -82,13 +97,14 @@ export function InternalUserEdit(props: InternalUserEditDeps) {
         try {
           const user = await getUserDetail(props.coreStart.http, props.sourceUserName);
           setAttributes(buildAttributeState(user.attributes));
+          setBackendRoles(user.backend_roles);
           if (action === 'edit') {
             setUserName(props.sourceUserName);
           } else {
             setUserName(props.sourceUserName + '_copy');
           }
         } catch (e) {
-          addToast(createErrorToast('fetchUser', 'load data'));
+          addToast(createUnknownErrorToast('fetchUser', 'load data'));
```

Comment:
Line 137 has a check about e.message before using calling it unknown error. Shall we do the same here?

Also createUnknownErrorToast seems actually mean createErrorToastWithoutErrorMessage. Leave it up to you to decide.

Reply:
L137 is because we know the server may return error 'Password needs to be 8 chars long and ...'

Not sure if any known error here.